content/doc/book/managing/system-properties.adoc

```diff
@@ -385,6 +583,52 @@ dd {
     **Since:** 2.37 +
     **Description:** Timeout for each https://wiki.jenkins.io/display/JENKINS/Ping+Thread[ping between the master and slaves], in seconds
 
+`hudson.slaves.ConnectionActivityMonitor.enabled`::
+// This looks like a dead feature? Introduced 2011 and disabled by default?
```

Comment:
Would be great to submit a follow-up ticket. I am not sure we want to keep it, Remoting monitoring has improved a lot since 2011. CC @jeffret-b